### PR TITLE
Exposed interceptors property.

### PR DESCRIPTION
**Pull request by matracey**
Added an interceptors property to ODataClient which exposes the underlying axios interceptors property. This is useful for setting pre- and post-request handlers.

### CODE REVIEW

File: lib/ODataClient.d.ts [hunk 1]
```diff
@@ -5,6 +5,10 @@ export declare class ODataClient {
     defaultRequestConfig: AxiosRequestConfig;
     private httpClient;
     constructor(baseUrl: string, config?: AxiosRequestConfig);
+    readonly interceptors: {
+        request: import("axios").AxiosInterceptorManager<AxiosRequestConfig>;
```
Comment (zhevron) [inline lib/ODataClient.d.ts]: Would rather have the types imported at the top of the file instead of inline imports.

File: lib/ODataClient.d.ts [hunk 2]
```diff
@@ -5,6 +5,10 @@ export declare class ODataClient {
...
+        response: import("axios").AxiosInterceptorManager<import("axios").AxiosResponse<any>>;
```
Comment (zhevron) [inline lib/ODataClient.d.ts]: Would rather have the types imported at the top of the file instead of inline imports.

**Review by zhevron — CHANGES_REQUESTED** [review]
Sorry for taking so long in getting to this. Thanks for submitting this! Just a couple of small fixes and it looks good to me.

**Comment by matracey** [comment]
Sorry it took me so long to get back to you. I missed notifications on this! I'm honestly not sure how I managed to go with inline Imports - I'd always favor having them at the top as well. I'll push a fix for this.

**Comment by matracey** [comment]
Ok. I've made the change. The `.d.ts` file is auto-generated, so I'm not sure why it decided to use the inline imports instead of using the existing import declaration at the top of the file, but I've fixed that now anyway.

**Review by zhevron — APPROVED** [review]

**Comment by zhevron** [comment]
Thanks for the PR! And I just noticed it does the same when I build locally. Guessing a TypeScript update caused it. Merged and published to npm.

**Pull request merged**